### PR TITLE
Fix checkstyle warnings

### PR DESCRIPTION
**Pull request by rolandpoulter**
@tldavies here are the code style fixes you asked for this morning.

### CODE REVIEW

File: lib/common/encryption.js [hunk 1]
```diff
@@ -99,7 +99,9 @@ function encryptionFactory(
         if(!signatures[algorithm]) {
             throw new TypeError('Unknown salt algorithm: ' + algorithm);
         }
-        if (salt && salt.charAt(0) === '$') salt = salt.split('$')[2];
+        if (salt && salt.charAt(0) === '$') {
+          salt = salt.split('$')[2];
```
Comment (anhou) [inline lib/common/encryption.js]: also style, 4X blanks before 'salt = salt.split('$')[2];'

**Comment by tldavies** [comment]
+1

**Comment by tldavies** [comment]
@benbp

**Comment by benbp** [comment]
:+1:

**Comment by jlongever** [comment]
was that travis-ci build failure resolved?

**Comment by rolandpoulter** [comment]
Pretty sure that failure is not related to the code style fixes. Most likely a false negative.

**Comment by rolandpoulter** [comment]
Rebased.

**Comment by tldavies** [comment]
@RackHD/corecommitters Please merge :-)

**Pull request merged**